pkg/backup/item_backupper.go

```diff
@@ -415,9 +420,14 @@ func (ib *defaultItemBackupper) takePVSnapshot(obj runtime.Unstructured, log log
 		}
 	}
 
-	pvFailureDomainZone := pv.Labels[zoneLabel]
-	if pvFailureDomainZone == "" {
-		log.Infof("label %q is not present on PersistentVolume", zoneLabel)
+	// TODO: prefer stable labels for region in v1.18
+	pvFailureDomainZone, labelFound := pv.Labels[zoneLabel]
+	if !labelFound {
```

Comment:
This should check for the `zoneLabelStable` first, then try to fallback to the deprecated label if it isn't there. It makes the deprecation intention more visible.

Reply:
Now that I think about it: maybe `zoneLabelStable` should be changed to just `zoneLabel`, and the existing `zoneLabel` could be called `zoneLabelDeprecated`. The intention here is to, when the old label is deprecated, there'd be no context for `zoneLabelStable` (stable vs. ?).

Reply:
thanks for your review @carlisia 
for the `zoneLabelStable` i used the same nomenclature as kubernetes codebase uses, but i can change that per your suggestion.

Reply:
done